package.json

```diff
@@ -3,36 +3,111 @@
   "version": "0.1.0",
   "description": "%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "prepare": "node ./scripts/prepare.js",
+    "lint": "eslint . --ignore-path=.gitignore --ext=ts",
+    "lint:fix": "npm run lint -- --fix",
+    "unit": "jest",
+    "test": "npm run lint && npm run unit",
     "coverage": "npm run unit -- --coverage",
-    "lint": "eslint . --ignore-path=.gitignore",
+    "clean": "rm -rf ./dist",
+    "check-types": "tsc --pretty --project tsconfig.json",
+    "compile": "npm run check-types && swc src -d dist",
+    "build": "npm run clean && npm run compile",
+    "dev": "swc src -d dist -w",
     "start": "lc39 dist/index.js",
-    "start:local": "npm start -- --env-path ./default.env",
-    "test": "npm run lint && npm run unit",
+    "start:local": "npm run compile && npm start -- --env-path ./default.env",
     "update-docker-version": "sed -i.bck \"s|version=\\\"[0-9]*\\.[0-9]*\\.[0-9]*.*\\\"|version=\\\"${npm_package_version}\\\"|\" Dockerfile",
     "update-changelog": "node ./scripts/update-changelog.js ${npm_package_version}",
-    "unit": "jest",
-    "version": "npm run update-changelog && npm run update-docker-version && rm -fr Dockerfile.bck && git add CHANGELOG.md Dockerfile",
-    "build": "tsc --project tsconfig.json"
+    "version": "npm run update-changelog && npm run docs && npm run update-docker-version && rm -fr Dockerfile.bck && git add CHANGELOG.md Dockerfile docs"
   },
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
-    "@mia-platform/custom-plugin-lib": "^4.3.2",
-    "fastify": "^3.25.1",
-    "typescript": "^4.5.4"
+    "@mia-platform/custom-plugin-lib": "^5.0.0"
   },
   "devDependencies": {
     "@mia-platform/eslint-config-mia": "^3.0.0",
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.194",
+    "@swc/jest": "^0.2.21",
     "@types/jest": "^27.0.3",
+    "@typescript-eslint/eslint-plugin": "^5.26.0",
+    "@typescript-eslint/parser": "^5.26.0",
     "eslint": "^8.5.0",
-    "jest": "^27.4.5",
-    "ts-jest": "^27.1.2"
+    "eslint-plugin-jest": "^26.4.4",
+    "husky": "^8.0.1",
+    "jest": "^28.1.0",
+    "json-schema-to-ts": "^2.5.2",
+    "ts-jest": "^28.0.3",
+    "typescript": "^4.7.2"
+  },
+  "peerDependencies": {
+    "@mia-platform/lc39": "^6.0.0"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=16"
   },
   "private": true,
   "eslintConfig": {
-    "extends": "@mia-platform/eslint-config-mia"
+    "extends": [
```

Comment:
can we use an .eslintrc file to prevent getting a huge package.json?

Reply:
Moved `eslint` configuration into its own file.